cnxarchive/views/sitemap.py

```diff
@@ -77,3 +81,32 @@ def sitemap(request):
     resp.content_type = 'text/xml'
     resp.body = xml()
     return resp
+
+
+@view_config(route_name='sitemap-index', request_method='GET')
+def sitemap_index(request):
+    """Return a sitemap index xml file for search engines."""
+    sitemaps = []
+    with db_connect() as db_connection:
+        with db_connection.cursor() as cursor:
+            cursor.execute("""\
+                SELECT t.*
+                FROM (
+                    SELECT module_ident,
+                           row_number() OVER (ORDER BY module_ident) AS row
+                    FROM latest_modules
+                    WHERE portal_type NOT IN (
+                        'CompositeModule', 'SubCollection')
+                ) AS t
+                WHERE t.row % {} = 1""".format(SITEMAP_LIMIT))
```

Comment:
This could have been written as a CTE (`WITH t(module_ident, row) AS ( …`) which is usually a bit clearer, though in this case, it's pretty much equivalent, in readability and performance. Usually not a fan of `*` in production code, since it can hide bugs when a table changes, but this is a defined subquery, so changes would have to be right here, anyway, so that gets a pass. :-) Ah, but I see we don't use all the values, so just return the module_ident, please.

Reply:
That was confusing -  the only call for action in there is to get rid of the ignored `_` return
This is just a general cleanup - avoids an message from flake8 for me, but could go out  as is.

Reply:
That's done.